### PR TITLE
feat(gh-actions): build and test arns app

### PR DESCRIPTION
**Pull request by atticusofsparta**
actions setup to build and test on pushes to any branch

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Build and Test
+
+on:
+  push:
+    branches: ['**']
```
Comment (dtfiedler) [inline .github/workflows/build_and_test.yml:5]: not needed - you remove this whole line and just have it as `push`

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+jobs:
+  test_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (dtfiedler) [inline .github/workflows/build_and_test.yml:11]: Let's use v3

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (dtfiedler) [inline dist/index.html:2]: Please add `dist` to .gitignore

**Review by dtfiedler — CHANGES_REQUESTED** [review]

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (atticusofsparta) [reply .github/workflows/build_and_test.yml]: add all requested changes in commit 0b1d5327776ed08bb564010ec564045a84dfde2e

**Review by dtfiedler — APPROVED** [review]
looks good, think we got the jest stuff sorted. we'll probably want to add prettier checks in here to keep the codebase clean. could be a separate action that we only check on merges

**Pull request merged**